1-js/02-first-steps/17-javascript-specials/article.md

```diff
@@ -1,234 +1,238 @@
-# JavaScript specials
+# JavaScriptowe detale
 
-This chapter briefly recaps the features of JavaScript that we've learned by now, paying special attention to subtle moments.
+Ten rozdział pokrótce podsumowuje cechy JavaScriptu, których nauczyliśmy się do tej pory, zwracając szczególną uwagę na drobne akcenty.
 
-## Code structure
+## Struktura kodu
 
-Statements are delimited with a semicolon:
+Instrukcje są rozdzielane średnikiem:
 
 ```js run no-beautify
-alert('Hello'); alert('World');
+alert('Witaj'); alert('Świecie');
 ```
 
-Usually, a line-break is also treated as a delimiter, so that would also work:
+Zwykle znak końca wiersza jest również traktowany jako separator, więc to działałby również:
 
 ```js run no-beautify
-alert('Hello')
-alert('World')
+alert('Witaj')
+alert('Świecie')
 ```
 
-That's called "automatic semicolon insertion". Sometimes it doesn't work, for instance:
+Nazywa się to „automatycznym wstawianiem średnika”. Czasami to jednak nie działa, na przykład:
 
 ```js run
-alert("There will be an error after this message")
+alert("Po tej wiadomości wystąpi błąd")
 
 [1, 2].forEach(alert)
 ```
 
-Most codestyle guides agree that we should put a semicolon after each statement.
+Większość wytycznych do stylów kodowania (ang. *codestyle guides*) twierdzi, że po każdym wyrażeniu należy umieścić średnik.
```

Comment:
Please do not add any phrases that weren't originally in sentence.
I'd also consider changing "Większość wytycznych do stylów kodowania" for "Większość styli kodowania" as it changes the original meaning IMO.